<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Title for MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_status">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>971</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="title">
     <string>REAL TIME STATUS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="cur_expected_vop_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>338</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Cur expected vop:</string>
     </property>
    </widget>
    <widget class="QLabel" name="framerate_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>320</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Mipi frame rate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_module_type_value">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>50</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Spot</string>
     </property>
    </widget>
    <widget class="QLabel" name="framerate_value">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>170</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>xxx fps</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_exp_pvdd_value">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>135</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>0 V</string>
     </property>
    </widget>
    <widget class="QLabel" name="strmTimeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>320</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Streaming time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_measurement_value">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>170</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Full</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_temperature_value">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>130</y>
       <width>135</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>32</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_work_mode_value">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>90</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>PCM</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_module_type_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>50</y>
       <width>320</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Cur module type:</string>
     </property>
    </widget>
    <widget class="QLabel" name="pvddLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>338</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Cur expected pvdd:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_environment_value">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>130</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Indoor</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_exp_vop_value">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>135</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>-25 V</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_measurement_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>170</y>
       <width>338</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Cur measurement:</string>
     </property>
    </widget>
    <widget class="QLabel" name="strmTime_value">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>170</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>12:34:56</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_work_mode_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>90</y>
       <width>320</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Cur work mode:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_powermode_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>210</y>
       <width>338</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Cur power mode:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_temperature_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>338</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Inside temperature:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_environment_label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>130</y>
       <width>338</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Cur environment:</string>
     </property>
    </widget>
    <widget class="QLabel" name="cur_powermode_value">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>210</y>
       <width>140</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>NA</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="rtcTime">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>0</y>
       <width>240</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="depth_view">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>840</width>
      <height>640</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>37</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Device is not ready yet.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>0</y>
      <width>531</width>
      <height>871</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_dtof_wkmode">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Work mode for dToF</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_dtof_wkmode">
        <item>
         <widget class="QRadioButton" name="radioButton_fhr">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>FHR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_pcm">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PCM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_phr">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PHR</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_rgb_wkmode">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Work mode for RGB</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_rgb_wkmode">
        <item>
         <widget class="QRadioButton" name="radioButton_yuyv">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>YUYV</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_nv12">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>NV12</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_dtof_environment">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Environment type for dToF</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_dtof_environment">
        <item>
         <widget class="QRadioButton" name="radioButton_indoor">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>INDOOR</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_outdoor">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>OUTDOOR</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_dtof_framerate">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Frame rate (fps):</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_dtof_framerate">
        <item>
         <widget class="QRadioButton" name="radioButton_15fps">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>15</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_25fps">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_30fps">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>30</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_60fps">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>60</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_dtof_powermode">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Power mode for dToF</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_dtof_powermode">
        <item>
         <widget class="QRadioButton" name="radioButton_div1">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Div1</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_div2">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Div2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_div3">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Div3</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="confidence_view">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>320</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Confidence Bitmap is NA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_push_button">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_pushButtons">
        <property name="spacing">
         <number>37</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="startStopButton">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quitButton">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
    <zorder>groupBox_push_button</zorder>
    <zorder>groupBox_dtof_powermode</zorder>
    <zorder>groupBox_dtof_wkmode</zorder>
    <zorder>groupBox_rgb_wkmode</zorder>
    <zorder>groupBox_dtof_environment</zorder>
    <zorder>groupBox_dtof_framerate</zorder>
    <zorder>confidence_view</zorder>
   </widget>
   <widget class="QLabel" name="AutoTestTimes_value">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>890</y>
      <width>135</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QLabel" name="AutoTestTimes_label">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>890</y>
      <width>338</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Auto test times:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
